owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/authentication/OCAuthenticationViewModelTest.kt

```diff
@@ -160,7 +161,13 @@ class OCAuthenticationViewModelTest : ViewModelTest() {
     fun loginOAuthOk() {
         every { loginOAuthAsyncUseCase.execute(any()) } returns UseCaseResult.Success(OC_BASIC_USERNAME)
         ocAuthenticationViewModel.loginOAuth(
-            OC_BASIC_USERNAME, OC_AUTH_TOKEN_TYPE, OC_ACCESS_TOKEN, OC_REFRESH_TOKEN, OC_SCOPE, null
+            username = OC_BASIC_USERNAME,
+            authTokenType = OC_AUTH_TOKEN_TYPE,
+            accessToken = OC_ACCESS_TOKEN,
+            refreshToken = OC_REFRESH_TOKEN,
+            scope = OC_SCOPE,
+            updateAccountWithUsername = null,
```

Comment:
Could we initialise `updateAccountWithUsername` to null in `loginOAuth` method and remove this line?

Reply:
Yes, thanks 🍻